programs/keeper/CMakeLists.txt

```diff
@@ -127,7 +132,9 @@ if (BUILD_STANDALONE_KEEPER)
 
         ${LINK_RESOURCE_LIB_STANDALONE_KEEPER}
     )
-
+    if (TARGET ch_contrib::qpl)
```

Comment:
Clickhouse-keeper is an implementation of the RAFT protocol. It is different from the ClickHouse database. No need to link ClickHouse keeper against QPL.